src/components/WeekDayPicker/readme.md

```diff
@@ -0,0 +1,175 @@
+##### WeekDayPicker base:
+
+```js
+import React, { useState } from 'react';
+import { WeekDayPicker } from 'react-rainbow-components';
+
+const SimpleWeekDayPicker = (props) => {
+    const [day, setDay] = useState('');
+
+    const handleOnChange = e => {
+        const weekDay = e.currentTarget.value;
```

Comment:
As declared in the interface this component is meant to send here a string with 'monday', 'tuesday', .... or an array of theses string values.